### PR TITLE
WIP: Log configuration in KafkaConsumerActor preStart

### PR DESCRIPTION
**Pull request by muditsin**
Fixes: https://github.com/akka/alpakka-kafka/issues/609

### CODE REVIEW

**Comment by ennru** [comment]
Thanks for looking into this. We'd need to use `ConsumerSettings.toString` instead, as the settings can be tweaked in the code.
And it must be wrapped with `if (log.isDebugEnabled)`.

**Comment by muditsin** [comment]
@ennru, my pleasure! :) Updated based on your comment. Is there a test to add for this? If so, how would I go about it?

File: core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala [hunk 1]
```diff
@@ -315,6 +315,8 @@ import scala.util.control.{NoStackTrace, NonFatal}
   override def preStart(): Unit = {
     super.preStart()
     try {
+      if (log.isDebugEnabled)
+        log.debug(settings.toString)
```
Comment (ennru) [inline core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala]: Some note "creating Kafka consumer with" would be great.

Comment (muditsin) [reply core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala]: @ennru Updated

**Comment by muditsin** [comment]
> And it must be wrapped with `if (log.isDebugEnabled)`.
Although I have added it, shouldn't the logger take care of something like the `isDebugEnabled`?

**Comment by ennru** [comment]
Yes, it won't log the message, but the `toString` would still be called before it detects that it doesn't need the `String`.

**Comment by ennru** [comment]
Thank you!

**Pull request merged**